src/main/java/uk/gov/hmcts/reform/pip/data/management/service/AccountManagementService.java

```diff
@@ -42,4 +47,32 @@ public Boolean getIsAuthorised(UUID userId, ListType listType, Sensitivity sensi
         }
     }
 
+    public List<String> getAllAccounts(String provenances, String role)
+        throws JsonProcessingException  {
+        try {
+            String result = webClient.get().uri(String.format(
+                    "%s/account/all?provenances=%s&roles=%s", url, provenances, role))
```

Comment:
Does this call works actually? I can see that account-management endpoint required a list of user provenance and a list of role. You are passing in strings here :)

![image](https://user-images.githubusercontent.com/80688761/205317767-5dd74aef-0681-4777-a1aa-f52b1f9a6691.png)

Reply:
yes, this is working and tested as well :)